<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить кино</string>
  </property>
  <widget class="QLineEdit" name="genre">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>133</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="setFilePici">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>55</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выбрать файл</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameCinema">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>13</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="description">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>218</y>
     <width>351</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="textNameCinema">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название кино</string>
   </property>
  </widget>
  <widget class="QLabel" name="textGenre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Жанр</string>
   </property>
  </widget>
  <widget class="QLabel" name="pici">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Изображение</string>
   </property>
  </widget>
  <widget class="QLabel" name="textDescription">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Описание</string>
   </property>
  </widget>
  <widget class="QLabel" name="textYear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>31</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Год</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="year">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>171</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1900</number>
   </property>
   <property name="maximum">
    <number>2022</number>
   </property>
   <property name="value">
    <number>2022</number>
   </property>
  </widget>
  <widget class="QLabel" name="textShortFilePici">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*файл не выбран</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAddCinema">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить кино</string>
   </property>
  </widget>
  <widget class="QLabel" name="textShortFileCinema">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>95</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*файл не выбран</string>
   </property>
  </widget>
  <widget class="QLabel" name="fileCinema">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>95</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Файл кино</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setFileCinema">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выбрать файл</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
